torch/_dynamo/variables/builder.py

```diff
@@ -1151,7 +1151,9 @@ def wrap_tensor(self, value: torch.Tensor):
         ):
             unimplemented("torch.compile does not support strided NestedTensor")
 
-        if is_sparse_any(value):
+        # Reject sparse, but not coo.
+        # TODO: remove this altogether when non-coo sparsity propagation is ready
```

Comment:
@pearu I noticed that removing this COMPLETELY (which I initially expected to have no impact), actually causes failures for the non-coo cases. I suspect we have a list somewhere of expected assertions to be thrown during the tests for dynamo, for example:

PYTORCH_TEST_WITH_DYNAMO=1 python test/test_sparse_csr.py -k test_add_SparseCSC_cpu_float32

However, I am not sure yet WHERE and HOW that list is defined? So for now, I just kept the unimplemented for non-COO